### PR TITLE
adds validation to file before uploading

### LINKED ISSUE #436: no validation of file before uploading
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
there is no validation for the file to be uploaded
on `events/import` and `admin/content/translations`

**Expected behavior:**
there should be proper validation for the file for allowed extensions

### PR DESCRIPTION
**Pull request by anu-007**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
adds validation to file before uploading

#### Changes proposed in this pull request:

-added validations to uploaded file format

demo-https://open-event-frontend-branch6.herokuapp.com/events/import

Fixes #436

### CODE REVIEW

File: app/components/events/event-import-section.js [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+        file: {
+          identifier : 'file',
+          rules      : [
+            {
+              type   : 'empty',
+              prompt : this.l10n.t('Please upload a file')
+            },
+            {
+              type   : 'regExp',
+              value  : '/^(.*.((zip|xml|ical)$))?[^.]*$/i',
```
Comment (niranjan94) [inline app/components/events/event-import-section.js]: Also add `.ics` and `.xcal`  extensions

**Comment by anu-007** [comment]
@niranjan94 added `.ics` and `.xcal` extensions and updated UI.

File: app/templates/components/events/event-import-section.hbs [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <div class="sixteen wide column">
-  <form class="ui form">
+  <form class="ui form" {{action 'submit' on='submit'}}>
     <div class="field {{if device.isMobile 'sixteen' 'five'}} wide column">
-      <label class='required'> {{t 'Select event source file to import'}}</label>
-      {{input type='file'}}
+      <label class='required' for='file'> {{t 'Select event source file to import'}}</label>
```
Comment (sumedh123) [inline app/templates/components/events/event-import-section.hbs]: Double quotes here.

**Comment by sumedh123** [comment]
@anu-007 Also see the tooltip of the validation below:
eventsimport
eventsimport4
It would look better if they appear to the right side of the input field.

**Comment by anu-007** [comment]
@sumedh123 done some changes ,please review.
image

**Pull request merged**